db/migrate/20210627155647_add_call_leg_to_call_data_records.rb

```diff
@@ -0,0 +1,6 @@
+class AddCallLegToCallDataRecords < ActiveRecord::Migration[6.1]
+  def change
+    add_column :call_data_records, :call_leg, :string, default: "A", null: false
```

Comment:
Should we put a default here? It seems like we're explicit anyway when we create the CDR. Maybe we can just update all to A and have no default

Reply:
We can remove it here and have the default in enumerize instead.